yb-voyager/cmd/import.go

```diff
@@ -65,6 +66,8 @@ func validateImportFlags() {
 	}
 	validateTableListFlag(target.TableList, "table-list")
 	validateTableListFlag(target.ExcludeTableList, "exclude-table-list")
+	validateImportObjectsFlag(target.ImportObjects, "object-list")
+	validateImportObjectsFlag(target.ExcludeImportObjects, "exclude-object-list")
```

Comment:
Should we add a check in case both `import-objects` and `exclude-import-objects` flag are provided. Like you did for table list.